### PR TITLE
[Concept Entry] PHP: Math functions

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

Adding a Math Functions concept to PHP as a base for new good first issues for various math functions. Added an `abs()` entry so the list of functions isn't completely empty when we merge this.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/math-functions/terms/abs/abs.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+$result = abs(n);
+```
+
+Where `n` is the number to find the absolute value of.
+
+## Example
+
+Using `abs()` to return the absolute value of `-6.5`:
+
+```php
```
Comment (KTom101) [inline content/php/concepts/math-functions/terms/abs/abs.md:30]: Can we change this to a codebyte example?

**Review by KTom101 — COMMENTED** [review]
@SSwiniarski I only have one suggestion.

**Comment by SSwiniarski** [comment]
@KTom101, done.

**Review by KTom101 — APPROVED** [review]
@SSwiniarski Ready for 2nd review!

**Review by HishamT — APPROVED** [review]
LGTM!

**Comment by yangc95** [comment]
@SSwiniarski live links: PHP [Math Functions](https://www.codecademy.com/resources/docs/php/math-functions): [abs()](https://www.codecademy.com/resources/docs/php/math-functions/abs)

cc: @KTom101

**Pull request merged**